### ISSUE TITLE
screen reader doesn't read the integrated terminal output 

### ISSUE THREAD

**Issue by adil-s** [Microsoft/vscode#48914]
I'm using NVDA version 2018.1 and when I choose run in terminal from context menu. it brings up the terminal. but doesn't read anything. it just says multi line edit. neither I am able to type anything in edit box nor review the output.

**Comment by Tyriar** [comment]
Have you turned on accessibility mode in the editor? (alt+f1)

**Comment by Tyriar** [comment]
@adil-s what version of VS Code are you using?

**Comment by adil-s** [comment]
Version 1.23.0-insider
Commit d801331ca874c03372fccc804c089e234e771b2b
Date 2018-04-28T09:46:13.844Z
Shell 1.7.12
Renderer 58.0.3029.110
Node 7.9.0
Architecture ia32

**Comment by adil-s** [comment]
Please look at this accessibility bug.

**Comment by Tyriar** [comment]
I just updated to NVDA 2018.1 and it seems to work fine, announcing the prompt and then any input/output that follows. Pressing esc to enter navigation mode to navigation/read previous output also works.
